### ISSUE TITLE
FFX_FSR2_OPTION_UPSAMPLE_SAMPLERS_USE_DATA_HALF option is not working

### ISSUE THREAD

**Issue by Bonefest** [GPUOpen-Effects/FidelityFX-FSR2#72]
Hi, I'm trying to compile the FSR 2.1 Using UE 4.26 on Windows with the FFX_FSR2_OPTION_UPSAMPLE_SAMPLERS_USE_DATA_HALF option enabled, but it produces a compile-time shader error related to the inclusion of the `ffx_fsr2_force16_begin.h` file. I see that the current version doesn't have that file at all. Could you please clarify what's going on, is it just an unfinished option or maybe I've missed something?

**Comment by starcid** [comment]
Yes, same to you, and it seems to be the most useful optimization compiler option., so hope it works.

**Comment by turanszkij** [comment]
I also have this problem, I couldn't compile FP16 mode. It breaks when `FFX_HALF` is also defined.
